contents/blog/open-source-business-models.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2021-04-30
+title: How We Monetized Our Open Source DevTool
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+By [Mo](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+So, you’ve decided to build an open-source product.
+
+While your choice is highly commendable, the rabid applause from devs on GitHub and HackerNews won’t pay the bills. At some point, you’ll need to monetize your product to sustain its development (and your team) in the long run.
```

Comment:
"rabid" kind of demeans our audience (as the folks on GitHub and HN), I get the point (and I like more out there tone like this), but I'd rephrase this

Reply:
very happy with being a little self demeaning - I just wouldn't apply it to our readers

Reply:
Changed to 'resounding'